morl_baselines/single_policy/ser/mo_q_learning.py

```diff
@@ -72,8 +69,7 @@ def __init__(
         self.learning_rate = learning_rate
         self.id = id
         self.seed = seed
-        if self.seed is not None:
-            seed_everything(self.seed)
+        self.np_random, _ = seeding.np_random(self.seed)
```

Comment:
Please write a small comment for why this case is different than other `MOAgent` (and in EUPG)